extensions/wikia/WikiaMaps/WikiaMapsHooks.class.php

```diff
@@ -104,9 +107,25 @@ public static function onBeforePageDisplay( $out ) {
 	 * @return bool
 	 */
 	public static function onAfterPageHeaderButtons( \Title $title, array &$buttons ): bool {
-		if ( $title->isSpecial( 'Maps' ) ) {
+		if ( self::isSpecialMapsPage() ) {
 			$label = wfMessage( 'wikia-interactive-maps-create-a-map' )->escaped();
 			$buttons[] = new Button( $label, '', '#', '', 'createMap' );
+
+			if ( self::isSingleMapPage() ) {
+				global $wgIntMapConfig;
+
+				$mapId = explode( '/', $title->getText() )[1]; // the page name (without ns prefix) is build like this: Maps/23414
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)